### PR TITLE
Batch statements

### PR DESCRIPTION
**Pull request by levydsa**
Depends on #12

### CODE REVIEW

File: libsql/src/main/kotlin/tech/turso/libsql/Connection.kt [hunk 1]
```diff
@@ -5,6 +5,8 @@ import tech.turso.libsql.proto.Value
 interface Connection : AutoCloseable {
     fun execute(sql: String)
 
+    fun executeBatch(sql: String)
```
Comment (haaawk) [inline libsql/src/main/kotlin/tech/turso/libsql/Connection.kt:8]: It might be nicer to accept a list of statements instead of a single string

Comment (haaawk) [reply libsql/src/main/kotlin/tech/turso/libsql/Connection.kt:8]: We probably also want parameters support in batch

Comment (levydsa) [reply libsql/src/main/kotlin/tech/turso/libsql/Connection.kt:8]: The Rust SDK doesn't support parameters in execute_batch yet

Comment (haaawk) [reply libsql/src/main/kotlin/tech/turso/libsql/Connection.kt:8]: Fair point. in such case, we're good here.

Comment (haaawk) [reply libsql/src/main/kotlin/tech/turso/libsql/Connection.kt:8]: A list of strings instead of ';' separated string is still better

Comment (haaawk) [reply libsql/src/main/kotlin/tech/turso/libsql/Connection.kt:8]: @levydsa ☝️

Comment (levydsa) [reply libsql/src/main/kotlin/tech/turso/libsql/Connection.kt:8]: why though? the user can just String.join themselves

**Review by haaawk — APPROVED** [review]

**Pull request merged**